apps/judicial-system/web/src/routes/Shared/Cases/PastCases.tsx

```diff
@@ -82,6 +84,15 @@ const DurationDate = ({ date }: { date: string | null }) => {
   )
 }
 
+const sortDefendants = (rowA: Row<Case>, rowB: Row<Case>) => {
```

Comment:
Added this as a quick fix for the defendant sorting, needs more refactoring later